weave/trace_server/calls_query_builder.py

```diff
@@ -427,11 +430,36 @@ def as_sql(self, pb: ParamBuilder, table_alias: str = "calls_merged") -> str:
             outer_query.limit = self.limit
             outer_query.offset = self.offset
 
+        # because we are selecting from a subquery, we need to prefix the fields, but we dont want to run all aggregations again
+        # inparticular argMaxMerge throws an error when run twice on display_name
+        # We also filter out summary_dump, because we add costs to summary dump in the select statement
+        final_select_fields = [
+            "all_calls." + field.field
+            if isinstance(field, CallsMergedAggField)
+            and field.agg_fn.__contains__("Merge")
```

Comment:
i think what you want to do here is something different. essentially all the aggregation resolution happens in your `all_calls` block. I think the cleaner solution is to simply rip through the fields, converting each to a plain `CallsMergedField` which will strip the aggs